javascript-carousel/main.js

```diff
@@ -0,0 +1,105 @@
+const $container = document.querySelector(".container");
+const $back = document.querySelector("#back");
+const $forward = document.querySelector("#forward");
+const $zero = document.querySelector("#zero");
+const $one = document.querySelector("#one");
+const $two = document.querySelector("#two");
+const $three = document.querySelector("#three");
+const $four = document.querySelector("#four");
+const $img = document.querySelector("#img");
+let counter = 0;
+const imgArr = ["images/001.png", "images/004.png", "images/007.png", "images/025.png",
+"images/039.png"];
+const $dotArr = [$zero, $one, $two, $three, $four];
+
+const changeDot = function() {
+  for (let i = 0; i < $dotArr.length; i++){
+    if (i === counter){
+      const black = $dotArr[i];
+      black.setAttribute("class", "fas fa-circle")
+    } else {
+      const white = $dotArr[i];
+      white.setAttribute("class", "far fa-circle")
+    }
+  }
+}
+
+const timer = function() {
+  if (counter === 4){
+    counter = 0;
+  } else {
+    counter++;
+  }
+  changeDot();
+  $img.setAttribute("src", imgArr[counter])
+}
+
+setInterval(timer, 3000);
```

Comment:
The interval should reset whenever the user interacts with the carousel.

Reply:
resolved